easybuild/easyblocks/v/vmd.py

```diff
@@ -45,6 +46,15 @@
 class EB_VMD(ConfigureMake):
     """Easyblock for building and installing VMD"""
 
+    def extract_step(self):
+        """Custom extract step for VMD."""
+        super(EB_VMD, self).extract_step()
+
+        vmddir = os.path.join(self.src[0]['finalpath'], '%s-%s' % (self.name.lower(), self.version))
```

Comment:
@akesandgren We should determine this once, and save it in `self.vmd_dir`, since we're using the same thing also in other places. You can probably also use:

```python
self.vmd_dir = os.path.join(self.builddir, '%s-%s' % (self.name.lower(), self.version))
```

We should set this in `__init__`, to make sure it's *always* defined (also under `--module-only` for example)

Reply:
Can i use self.builddir in the extract step? I thought it didn't setup until later.

Reply:
Found an example of using self.builddir in __init__ at least (Matlab) so should be doable.
But start_dir vs builddir and the above self.src[0]['finalpath'], I can never remember which is what...

Reply:
You can use `self.builddir` early on, as soon as the easyblock is initialized (it's defined in `EasyBlock.__init__`).

`self.builddir` is the top-level build directory.
`self.start_dir` is the build directory in which the build will be performed by default. This is typically the unpack source directory (so it's under `self.builddir`). But if you have multiple source files, it may be equal to `self.builddir`.
`self.src[0]['finalpath']` is the location where the first source file was unpacked. If there's only one source file, it's the same as `self.start_dir`